Rakefile

```diff
@@ -30,5 +30,16 @@ namespace :db do
   task :environment do
     require_relative 'application'
   end
-end
 
+  # To correct for https://github.com/Safecast/ingest/pull/2/files#diff-5835da67485dd547fb9545b67446cc53R7
+  # Can remove after being deployed & run on prd env
+  desc 'JSON parse any string measurement payloads'
+  task fix_payload_escaping: :environment do
+    Measurement.all.each do |m|
+      if m.payload.is_a? String
+        m.payload = JSON.parse(m.payload)
```

Comment:
Sounds good. Added in https://github.com/Safecast/ingest/pull/4/commits/537767ff8f62cab4a5d42db6559e46274fcade69